cfgov/unprocessed/css/activity.less

```diff
@@ -37,31 +37,31 @@
           aside.activity
             h1.h4
               .cf-icon
-            ul.list.list__unstyled.list__spaced
-              li.list_item
-                a.list_link
+            ul.m-list.m-list__unstyled.m-list__spaced
+              li.m-list_item
+                a.m-list_link
   tags:
     - cfgov-activity
 */
 
 .activity {
     margin-bottom: unit(30px / @base-font-size-px, em);
 
-    .list_link {
+    .m-list_link {
         .u-link__colors(@pacific, @pacific, @pacific-60, @pacific, @navy);
     }
 
-    .date {
+    .a-date {
         white-space: nowrap;
     }
 
     .respond-to-min(@bp-sm-min, {
-        .list_link {
+        .m-list_link {
             &:after {
                 content: '';
```

Comment:
Hmmm I see it, it appears on the items in the sidebar on /about-us/, but not sure why it's there. I removed it and couldn't see a difference. I looked at the file history too and it looks old as it predates our reorganization of the less files, so doesn't have the history of who created it originally. Probably a candidate for removal, but probably best left to another PR.